tests/FakeItEasy.Tests/Creation/ProxyGeneratorSelectorTests.cs

```diff
@@ -92,16 +93,15 @@ public void Should_delegate_calls_to_default_generator_asking_about_non_delegate
             var getHashCode = fake.GetType().GetMethod("GetHashCode");
 
             string reason = null;
-            A.CallTo(() => this.defaultProxyGenerator.MethodCanBeInterceptedOnInstance(getHashCode, fake, out reason))
-                .Returns(true).AssignsOutAndRefParameters("reason");
+            A.CallTo(() => this.defaultProxyGenerator.MethodCanBeInterceptedOnInstance(getHashCode, fake, out reason)).Returns(true).AssignsOutAndRefParameters("reason");

```

Comment:
I see this:

![image](https://cloud.githubusercontent.com/assets/677704/14558116/e8743802-0300-11e6-9da8-21bb2a9e550c.png)

1920, 100% scaling, all browser (Chrome) defaults, no Github.ExpandinizR